pom.xml

```diff
@@ -214,10 +214,9 @@
         <httpclient.version>4.5.6</httpclient.version>
         <httpcore.version>4.4.10</httpcore.version>
         <httpmime.version>${httpclient.version}</httpmime.version>
-        <jackson2-version>2.9.8</jackson2-version>
         <jai-imageio-core.version>1.3.1</jai-imageio-core.version>
         <JavaAPIforKml.version>2.2.1_CODICE_1</JavaAPIforKml.version>
-        <jackson.version>2.10.0</jackson.version>
```

Comment:
Why are we not upgrading to 2.10.1?

Reply:
Copying from the thirdparty PR:

I chose 2.9.10 for two reasons.

The first is because it addresses all the CVEs that are present up to and including 2.9.8, just the same as 2.10.1.

The second is because this is a much simpler update given it is a patch version. I initially spent a day doing the third party update to 2.10.1. This involved a lot of indivudal pom changes in the felix plugin that were only addressable, individually, after building thirdparty and then building DDF again. You only get one dependency error at a time, so after ~30 line changes, third party build, and DDF build, I eventually went to see if a patch version would get us there quicker.

If the goal is to get the CVEs down in Jackson ASAP, this is the right path. If the goal is to upgrade 2.10.1, then I can take these PRs down and continue that work.

@clockard and @emmberk let me know your thoughts.

Reply:
See https://github.com/codice/thirdparty/pull/44#discussion_r352740345